freestyle/shared/src/main/scala/freestyle/Implicits.scala

```diff
@@ -24,4 +24,9 @@ trait Interpreters {
     Inject.catsFreeRightInjectInstance(I.value)
 }
 
-object implicits extends Interpreters
+trait Monads {
```

Comment:
Can we rename this to `FreeSInstances`? down the road we may want to explicitly add others here.